functions/src/modules/SE.functions.ts

```diff
@@ -194,94 +194,92 @@ export const createTaskFromChunks = functions.database
     });
     return snapshot.val();
   });
-  return snapshot.val();
-});
 
 
 
 /**
  * Sends a notification email to the coordinator & udpates the corresponding Task
  */
-const seUploadsBucketUrl = functions.config().sound_editing.uploads.bucket_name;
-export const processUploadedFile = functions.storage.bucket(seUploadsBucketUrl).object()
-.onFinalize(async object => {
+const stoargeBaseDomain = functions.config().storage['root-domain'];
+export const processUploadedFile = functions.storage.bucket(`uploads${stoargeBaseDomain}`).object()
```

Comment:
`storageBaseDomain` will not have a dot in the beginning, it will be something like `test.audioseva.purebhakti.info`. So put the dot here in between.